cmd/clusterctl/api/v1alpha3/labels.go

```diff
@@ -30,6 +30,9 @@ const (
 	// Example: resources shared between instances of the same provider:  CRDs,
 	// ValidatingWebhookConfiguration, MutatingWebhookConfiguration, and so on.
 	ClusterctlResourceLifecyleLabelName = "clusterctl.cluster.x-k8s.io/lifecycle"
+
+	// ClusterctlForceMoveLabelName can be set on objects that providers wish to move that are not part of a cluster
+	ClusterctlForceMoveLabelName = "clusterctl.cluster.x-k8s.io/move"
```

Comment:
Since we renamed the label to `clusterctl.cluster.x-k8s.io/move` I don't think we need the `Force` terminology.
```suggestion
	// ClusterctlMoveLabelName can be set on objects that providers wish to move that are not part of a cluster
	ClusterctlMoveLabelName = "clusterctl.cluster.x-k8s.io/move"
```

Reply:
fixed in amended commit.